clients/java/client/src/main/java/org/camunda/bpm/client/task/impl/dto/SetVariablesRequestDto.java

```diff
@@ -0,0 +1,24 @@
+package org.camunda.bpm.client.task.impl.dto;
+
+
+import org.camunda.bpm.client.impl.RequestDto;
+import org.camunda.bpm.client.variable.impl.TypedValueField;
+
+import java.util.Map;
+
+/**
+ * @author Taha Arian
+ */
```

Comment:
```suggestion
```
Please don't use `author` Javadocs. This is against our [Java Coding Style Guide](https://github.com/camunda/camunda-bpm-platform/wiki/Coding-Style-Java).

Reply:
thanks, fixed